syne_tune/experiments/launchers/launch_remote_common.py

```diff
@@ -56,13 +59,16 @@ def sagemaker_estimator_args(
     max_run = (
         int(1.2 * benchmark.max_wallclock_time) if benchmark is not None else 3600 * 72
     )
+    dependencies = syne_tune.__path__
+    if source_dependencies is not None:
+        dependencies = dependencies + source_dependencies
     sm_args = dict(
         entry_point=entry_point.name,
         source_dir=str(entry_point.parent),
         checkpoint_s3_uri=checkpoint_s3_uri,
         max_run=max_run,
         role=get_execution_role(),
-        dependencies=syne_tune.__path__ + benchmarking.__path__,
+        dependencies=dependencies,
```

Comment:
Needed, because we want people to use their own source deps, even if `benchmarking` is not even there.